addons/event/static/src/scss/event.scss

```diff
@@ -30,6 +30,12 @@
             min-height: 95px;
         }
     }
+    .attendee_badge{
```

Comment:
same here, please use an usual prefix `o_event_registration_kanban_badge` for instance